unitTests/ui/OmniboxResultListTest.ts

```diff
@@ -20,6 +20,11 @@ export function OmniboxResultListTest() {
       results = FakeResults.createFakeResults();
     });
 
+    it("HTMLElement, doesn't have a DOMChild", async done => {
```

Comment:
Why does this test need to be async?

Reply:
I'll remove it, all the other test before had it so I put it too